sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -251,27 +253,31 @@ export class EducationProgramOfferingController extends BaseController {
    * access to their specific offerings only).
    * @param locationId location id.
    * @param programId program id.
+   * @param programYearId program year id.
+   * @param offeringIntensitySelectedByStudent offering intensity selected by student.
    * @query includeInActivePY, if includeInActivePY is true,
    * then consider both active and inactive program year.
    * @returns key/value pair list of programs for students.
    */
   @AllowAuthorizedParty(AuthorizedParties.institution)
   @HasLocationAccess("locationId")
   @Get(
-    "location/:locationId/education-program/:programId/program-year/:programYearId/offerings-list",
+    "location/:locationId/education-program/:programId/program-year/:programYearId/offeringIntensitySelectedByStudent/:offeringIntensitySelectedByStudent/offerings-list",
```

Comment:
I feel the naming offeringIntensitySelectedByStudent is really big, can we have something like selectedOfferingIntensity or something? already the route is big.